composer.json

```diff
@@ -8,6 +8,7 @@
         "email": "paolo.greppi@libpf.com"
     }],
     "require": {
+        "italia/spid-php-lib": "^0.20.0",
```

Comment:
This is not needed because it's included in the main directory.

Reply:
I am not sure it's a good idea to remove this: 'hard coding' a class to the repo whilst is being actively updated is dangerous and difficult to maintain (and scale). I like the idea of offering a self contained plugin but there are risks involved. We should find a way to offer both options, maybe automatically selecting from within the plugin the most recent class version.

Reply:
I'm sorry. I'm having trouble understanding. Could you clarify? Any future updates of the plugin need to be updated manually and pushed into WP.org. Whether that's through Composer or by move/replace, it doesn't matter. This only applies to the developer working on this plugin and does not do anything for the end user using this plugin.

Reply:
So there won't be any other channel of distribution for the plugin? Only a manual update into WP.org? IE: not download a .zip from github?

Reply:
There might be but doesn't make sense not to distribute it on WP.org. This makes it an even stronger case why we should not require Composer to use this plugin.